packages/playground/windows/Playground-win32 (Package)/Playground-win32 (Package).wapproj

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could we avoid having spaces in the folder and file names? Many tools will fail to work. Especially if they originally created for Unix-like OSes.

Reply:
@vmoroz  can you clarify what tools created for Unix might have to deal with a Windows app packaging project?